README.md

```diff
@@ -11,21 +14,51 @@ UML diagrams for the Project
 
 The falsk application for yummy-recipies
 
-#Prerequisites
+# Prerequisites
 
 python 3.6 or a later python version is required to run this app.
 
-# installing
+# Installing
 clone the repository
 
 FOR HTTPS
 https://github.com/mwaz/yummy-recipies
 
 FOR SSH
-
 git@github.com:mwaz/yummy-recipies.git
 
 # Change Directory into the project folder
 
 $ cd yummy-recipies
 
+# Create a virtual environment with Python 3.6
+
+$ virtualenv --python=python3.6 yourenvname
+
+# Activate the virtual environment you have just created
+
+$ source yourenvname/bin/activate
+
+# Install the application's dependencies from requirements.txt to the virtual environment
+
+$ (yourenvname) pip install -r requirements.txt
+
+# Set up Unit Test Environment
+
+run the following command to install nose unit testing environment:
+
+$ (yourenvname) pip install nose
```

Comment:
`(yourenvname)` should not be included. Instead, have steps for installing and setting up the virtual environment.